### PR TITLE
Improve quick setup and config consistency

### PR DESCRIPTION
**Pull request by nwittler**
## What
- Add a small test for signal generation from a quick setup.
- Loading noise experiments from config. Including some inconsistencies storing classical devices. 

## Why
Check for changes in the quick setup. Make tests compatible with graph mode namespaces.

## How
Compare generated signals to stored reference.

## Checklist
Please include and complete the following checklist. Your Pull Request is (in most cases) not ready for review until the following have been completed. You can create a draft PR while you are still completing the checklist. Check the [Contribution Guidelines](https://github.com/q-optimize/c3/blob/dev/CONTRIBUTING.md) for more details. You can mark an item as complete with the `- [x]` prefix

- [x] Tests - Added unit tests for new code, regression tests for bugs and updated the integration tests if required
- [x] Formatting & Linting - `black` and `flake8` have been used to ensure styling guidelines are met
- [ ] Type Annotations - All new code has been type annotated in the function signatures using type hints
- [ ] Docstrings - Docstrings have been provided for functions in the `numpydoc` style
- [ ] Documentation - The tutorial style documentation has been updated to explain changes & new features
- [ ] Notebooks - Example notebooks have been updated to incorporate changes and new features
- [ ] Changelog - A short note on this PR has been added to the Upcoming Release section

### CODE REVIEW

File: test/test_quick_setup.py [hunk 1]
```diff
@@ -23,3 +25,14 @@ def test_exp_quick_setup_freqs() -> None:
...
     offset = gate.comps["d1"]["gaussian"].params["freq_offset"].get_value()
     assert qubit_freq == carrier_freq + offset
+
+@pytest.mark.integration
+def test_generator() -> None:
+    gen_signal = pmap.generator.generate_signals(pmap.instructions["rx90p[0]"])
+    with open("test/quick_data.pickle", "rb") as quickfile:
+        test_data = pickle.load(quickfile)
+    np.testing.assert_allclose(gen_signal["d1"]["ts"], test_data["d1"]["ts"])
+    np.testing.assert_allclose(
```
Comment (lazyoracle) [inline test/test_quick_setup.py:36]: One possible standard we might want to enforce is that original test writers should provide explicit `atol` or `rtol` values for tests that use `assert_allclose` type functionality. These tolerances should be the maximum acceptable worst case figures. This ensures that we don't tamper with tests later on when introducing some new feature or fixing a bug, with the assumption that the original test writer already accounted for the worst case figures and a failing test indicates something other than precision issues.

**Review by lazyoracle — CHANGES_REQUESTED** [review]

File: test/test_quick_setup.py (hunk 1 above)
Comment (nwittler) [reply test/test_quick_setup.py:36]: Good idea. Done.

**Comment by lazyoracle** [comment]
Do you mind if I rebase and cleanup a bit of the git history in this PR and then force push the branch? 😆

**Comment by nwittler** [comment]
> Do you mind if I rebase and cleanup a bit of the git history in this PR and then force push the branch? 😆

Go ahead, that's magic to me.

**Review by lazyoracle — APPROVED** [review]

**Pull request merged**